common/quotas/rate_limiter_impl.go

```diff
@@ -47,73 +48,54 @@ var _ RateLimiter = (*RateLimiterImpl)(nil)
 // NewRateLimiter returns a new rate limiter that can handle dynamic
 // configuration updates
 func NewRateLimiter(newRate float64, newBurst int) *RateLimiterImpl {
```

Comment:
nit: plz be consistent re. the renaming: rate -> rps.

Is this renaming for fixing the lint error of shadowing import?

Reply:
Fixed to just be rps. Yeah it's part of fixing the shadowed import linter issue